### PR TITLE
duckduckgo topic

### PR DESCRIPTION
**Pull request by theel0ja**
### Thank you for contributing! Please confirm the following:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  
---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this topic or collection should be curated:

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/duckduckgo/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+topic: duckduckgo
+display_name: DuckDuckGo
+logo: duckduckgo.png
+short_description: DuckDuckGo is an Internet search engine that emphasizes protecting searchers' privacy and avoiding the filter bubble of personalized search results.
```
Comment (talsafran) [inline topics/duckduckgo/index.md]: We try to keep the short and long descriptions different –– it looks like the CI failed on two things:

```

  2) Failure:
topics::duckduckgo topic#test_0015_has a short_description that differs from the body [/home/travis/build/github/explore/test/topics_test.rb:173]:
body and short description should differ.
Expected "DuckDuckGo is an Internet search engine that emphasizes protecting searchers' privacy and avoiding the filter bubble of personalized search results." to not be equal to "DuckDuckGo is an Internet search engine that emphasizes protecting searchers' privacy and avoiding the filter bubble of personalized search results.".
  3) Failure:
topics::duckduckgo topic#test_0023_has a valid short_description [/home/travis/build/github/explore/test/topics_test.rb:279]:
must have a short_description no more than 130 characters (currently 148)
```

How would you feel about changing the `short_description` to the following:

```diff
+ short_description: DuckDuckGo is an Internet search engine that prioritizes privacy by not tracking users.
- short_description: DuckDuckGo is an Internet search engine that emphasizes protecting searchers' privacy and avoiding the filter bubble of personalized search results.
```

Comment (theel0ja) [reply topics/duckduckgo/index.md]: took yours.

Comment (theel0ja) [reply topics/duckduckgo/index.md]: Took your `short_description` suggestion. https://github.com/theel0ja/explore/commit/d1652f5c51fbc0a50c6bb52be10fad1b9ca3cefe

**Comment by talsafran** [comment]
@theel0ja thank you! Your contribution is [live](https://github.com/topics/duckduckgo) ✨

**Pull request merged**